src/hipscat/pixel_tree/negative_tree.py

```diff
@@ -0,0 +1,22 @@
+from typing import List
+
+from hipscat.pixel_math import HealpixPixel
+from hipscat.pixel_tree import PixelAlignmentType, align_trees
+from hipscat.pixel_tree.pixel_tree import PixelTree
+
+
+def compute_negative_tree_pixels(tree: PixelTree) -> List[HealpixPixel]:
```

Comment:
I would have liked that, but it would lead to circular imports. We need `align_trees` for this function now, which needs to be able to create a `PixelTree`. There might be a better way of organising everything that doesn't run into that issue though.